test/uk/gov/hmrc/trusts/connectors/DesConnectorSpec.scala

```diff
@@ -154,6 +154,7 @@ class DesConnectorSpec extends BaseConnectorSpec {
     }
   }//trustsmatch
 
+
   ".checkExistingEstate" should {
 
     "return Matched " when {
```

Comment:
"trusts data match with existing trusts" to "estate data match with existing estate"

Reply:
Complete